### PR TITLE
closing menu to refresh container state

### PR DESCRIPTION
**Pull request by sombriks**
this PR solves #39

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 *.zip
+.vscode
+node_modules
+package-lock.json
```
Comment (sombriks) [inline .gitignore:4]: should this get versioned?

Comment (rgolangh) [reply .gitignore:4]: no need, this is +1

File: extension.js
Comment (rgolangh) [inline extension.js:1]: can you fix the indentation to remain as the original? this is standard for gnome-extensions

Comment (rgolangh) [reply extension.js:1]: oh yeah the  linter action will guide you exactly on what is expected. hope this helps.

Comment (sombriks) [reply extension.js:1]: i think lint solved it, please check it again when convenient 😄

**Pull request closed without merging**